### ISSUE TITLE
Dataset has no validation set

### ISSUE THREAD

**Issue by dee-hub** [Abdulbaqqi/CALIFORNIA-HOUSING#1]
Your splitting of the dataset, didn't account for validation test.

You should do something like

X_valid = X_train[20% of the training data ] 
y_valid = y_train[20% of the training labels]

**Comment by CodebyBaki** [comment]
Okayy. I will do that now
